libs/vue/src/schematics/component/files/__name@toClassName__.vue.template

```diff
@@ -0,0 +1,17 @@
+<template>
+  <p>Welcome to <%= componentName %></p>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: '<%= componentName %>',
+});
+</script>
+
+<% if (styleExtension == 'css') { %>
+<style scoped></style>
+<% } else { %>
+<style scoped lang="<%= styleExtension %>"></style>
+<% } %>
```

Comment:
With the changes to `schematic.ts` this file needs to be renamed `__name__.vue.template`, and references to `componentName` and `styleExtension ` need to be changed to `name` and `style` respectively.